src/site/_data/authorsData.json

```diff
@@ -1680,5 +1680,19 @@
       "name": "Corel",
       "unit": "Online Graphics"
     }
+  }, 
+  "antoinebisch": {
+    "name": {
+      "given": "Antoine",
+      "family": "Bisch"
+    },
+    "github": "antoinebisch",
+    "country": "UK",
+    "org": {
+      "name": "Google"
+    },
+    "descriptions": {
+      "en": "Web Ecosystem Consultant"
+    }
```

Comment:
We need author images for Antoine:

![image](https://user-images.githubusercontent.com/4713486/101686937-9a794200-3a1e-11eb-997a-eb946dde2a91.png)

Reply:
Uploaded with another commit.